core/trino-main/src/main/java/io/trino/sql/planner/planprinter/TextRenderer.java

```diff
@@ -154,23 +155,33 @@ private String printStats(PlanRepresentation plan, NodeRepresentation node)
         return output.toString();
     }
 
-    private void printMetrics(StringBuilder output, String label, Function<BasicOperatorStats, Metrics> metricsGetter, PlanNodeStats stats)
+    private void printMetrics(StringBuilder output, String label, Function<BasicOperatorStats, Metrics> metricsGetter, PlanNodeStats stats, boolean basicOperatorStats)
```

Comment:
I'm not an expert here but `printMetrics` looks like a method that should only print generic `Metrics`. I wouldn't put the physical input data size here. Is there another way to print it? Maybe using something similar to `io.trino.sql.planner.planprinter.PlanNodeStats#planNodeInputDataSize`?

Reply:
This method prints `physical read time` when `verbose` is added to explain analyze, I've assumed that physical input data should be in the same place as this information. But there is a catch—we want to print this metric without `verbose`. Extending metric to add log level looked as overkill, same with matching by name.